html/index.html

```diff
@@ -2,15 +2,20 @@
 <html lang="en">
   <head>
     <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
I have found it at https://developers.google.com/web/fundamentals and it specify to use Internet Explorer 9 Standards mode.

We can drop this line if you prefer.